localflavor/md/validators.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.utils.translation import ugettext_lazy as _
+
+
+class MDIDNOFieldValidator(RegexValidator):
+    """
+    Validation for Moldavian IDNO.
+
+    .. versionadded:: 1.6
+    """
+
+    error_message = _('Enter a valid IDNO number.')
+
+    def __init__(self):
+        super(MDIDNOFieldValidator, self).__init__(regex='^\d{13}$',
+                                                   message=self.error_message)
+
+    def __call__(self, value):
+        super(MDIDNOFieldValidator, self).__call__(value)
+
+
+# https://en.wikipedia.org/wiki/Vehicle_registration_plates_of_Moldova
+class MDLicensePlateValidator(RegexValidator):
+    """
+    Validation for Moldavian License Plates.
+
+    .. versionadded:: 1.6
+    """
+
+    error_message = _('Enter a valid license plate.')
+
+    def __init__(self):
+        super(MDLicensePlateValidator, self).__init__(regex='^\d{9}$',
+                                                      message=self.error_message)
+
+    def __call__(self, value):
+        value = value.upper()
+        if not self._is_valid(value):
+            raise ValidationError(self.error_message)
+
+    def _is_valid(self, value):
+        return (
+            self._is_old_format(value) or
+            self._is_new_format(value) or
+            self._is_president_format(value) or
+            self._is_diplomatic_format(value) or
+            self._is_police_format(value) or
+            self._is_foreign_format(value) or
+            self._is_state_security_format(value) or
+            self._is_gov_format(value)
+        )
+
+    @staticmethod
+    def _is_old_format(value):
+        from localflavor.md.choices import REGION_CHOICES
```

Comment:
These should be imported at the top (e.g. `from .choices import REGION_CHOICES`). The same applies to the other local imports.